tools/RAiDER/cli/raider.py

```diff
@@ -399,7 +399,7 @@ def calcDelaysGUNW():
 
     p.add_argument(
         '-m', '--weather-model', default='HRRR', type=str,
-        choices=['None', 'HRRR', 'HRES', 'GMAO'], help='Weather model.'
+        choices=['None', 'HRRR', 'HRES', 'GMAO', 'ERA5'], help='Weather model.'
```

Comment:
```suggestion
        choices=['None'] + ALLOWED_MODELS, help='Weather model.'
```

Might be better to change this to the `ALLOWED_MODELS` constant from RAiDER:
https://github.com/dbekaert/RAiDER/blob/develop/tools/RAiDER/models/allowed.py#L1

**Note:** will also need to add 
```python
from RAiDER.models import ALLOWED_MODELS
``` 
to this file